conf/crukmi.config

```
@@ -26,23 +26,23 @@ process {
     maxRetries     = 3
 
     withLabel: process_single {
-        cpus   = 1
-        memory = 5
+        cpus   = { 1 * task.attempt }
+        memory = { 5 * task.attempt }
```

Hmm, this one doesn't have `.GB`?
Shall we make it consistent:
```suggestion
        memory = { 5.GB * task.attempt }
```